webapp-django/crashstats/crashstats/models.py

```diff
@@ -40,7 +40,7 @@
 from crashstats.base.utils import requests_retry_session
 
 
-logger = logging.getLogger('crashstats_models')
+logger = logging.getLogger('crashstats.models')
```

Comment:
Changing it from `crashstats_models` to `crashstats.models` means that it get covered by the existing configuration. Nothing else looks at `crashstats_models` and it's a webapp-specific bit of code, so this is no-risk.